docs/section6/Rule6-5.md

```diff
@@ -51,18 +51,19 @@
 
           - Get normalized space lighting schedule in P_RMR: `normalized_schedule_p = normalize_space_schedules(space_p.interior_lighting)`
 
-        - Check if automatic shutoff control is modeled in space during building closed hours (i.e. if lighting schedule hourly value in B_RMR is equal to P_RMR during building closed hours): `schedule_comparison_result = compare_schedules(normalized_schedule_b, normalized_schedule_p, building_open_schedule_b, -111)`  
+        - Check if automatic shutoff control is modeled in space during building closed hours (i.e. if lighting schedule hourly value in B_RMR is equal to P_RMR during building closed hours): `schedule_comparison_result = compare_schedules(normalized_schedule_p, normalized_schedule_b, building_open_schedule_b, -111)`  
 
           **Rule Assertion:**
 
-          - Case 1: For building closed hours, if lighting schedule hourly value in B_RMR is equal to P_RMR: `if schedule_comparison_result == "MATCH": PASS`  
+          - Case 1: For building closed hours, if lighting schedule hourly value in B_RMR is equal to P_RMR: `if schedule_comparison_result['TOTAL_HOURS_COMPARED'] == schedule_comparison_result['TOTAL_HOURS_MATCH']: PASS`  
 
-          - Case 2: Else: `else: CAUTION and return schedule_comparison_result`  
+          - Case 2: Else: `else: FAILED and return schedule_comparison_result['EFLH_DIFFERENCE']`  
```

Comment:
I think this should be another place where we show "pass" outcome if flag more_stringent_value_allowed=true. If EFLHp>EFLHb then result is off but conservative.

Reply:
Make sense, I address this part in the RDS.